selfdrive/sensord/tests/test_sensord.py

```diff
@@ -55,24 +97,137 @@ def setUpClass(cls):
 
   @with_processes(['sensord'])
   def test_sensors_present(self):
-    sensor_events = messaging.sub_sock("sensorEvents", timeout=0.1)
-
-    start_time_sec = time.time()
-    events = []
-    while time.time() - start_time_sec < TEST_TIMESPAN:
-      events += messaging.drain_sock(sensor_events)
-      time.sleep(0.01)
+    # verify correct sensors configuration
+    events = read_sensor_events(10)
 
     seen = set()
     for event in events:
       for measurement in event.sensorEvents:
-        # Filter out unset events
+        # filter unset events (bmx magn)
         if measurement.version == 0:
           continue
         seen.add((str(measurement.source), measurement.which()))
 
     self.assertIn(seen, SENSOR_CONFIGURATIONS)
 
+  @with_processes(['sensord'])
+  def test_lsm6ds3_100Hz(self):
```

Comment:
let's also add a test case to ensure interrupts aren't coming in after sensord dies